samples/caf_sensor_manager/README.rst

```diff
@@ -0,0 +1,76 @@
+.. _caf_sensor_manager_sample:
+
+CAF Sensor Manager
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The CAF Sensor Manager sample demonstrates the functionality of the :ref:`caf_sensor_manager`.
+It shows how to use the Sensor Manager to sample data, aggregate data and then receive them in packages.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840
+
+Overview
+********
+
+The sample:
+
+1. Provides simulated input data using sensor_sim and :ref:`caf_sensor_manager`.
+#. Aggregates data using :ref:`caf_sensor_data_aggregator`.
+#. When aggregator buffer is full sends data to workload_sim module.
+#. Simulates work using workload_sim.
+
+Configuration
+*************
+
+|config|
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/caf_sensor_manager`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, test it by performing the following steps:
+
+1. |connect_terminal|
+#. Reset the kit.
+#. Observe that output similar to the following is logged on UART:
+
+   .. parsed-literal::
+      :class: highlight
+
+      [00:12:*** Booting Zephyr OS build v2.7.99-ncs1-2182-gbb39dfe3121a  ***
+      [00:00:00.258,605] <inf> event_manager: e:module_state_event module:main state:READY
+      [00:00:00.258,728] <inf> event_manager: e:module_state_event module:sensor_sim_ctrl state:READY
+      [00:00:00.258,850] <inf> event_manager: e:sensor_state_event sensor:accel_sim_xyz state:ACTIVE
+      [00:00:00.258,941] <inf> event_manager: e:sensor_data_aggregator_event Send sensor buffer desc: accel_sim_xyz
+      [00:00:00.259,033] <inf> event_manager: e: sensor_data_aggregator_release_buffer_event
+      [00:00:00.259,124] <inf> event_manager: e:module_state_event module:sensor_manager state:READY
+      [00:00:00.458,526] <inf> event_manager: e:sensor_data_aggregator_event Send sensor buffer desc: accel_sim_xyz
+      [00:00:00.458,709] <inf> event_manager: e: sensor_data_aggregator_release_buffer_event
+      [00:00:00.658,538] <inf> event_manager: e:sensor_data_aggregator_event Send sensor buffer desc: accel_sim_xyz
+      [00:00:00.658,721] <inf> event_manager: e: sensor_data_aggregator_release_buffer_event
+
```

Comment:
Is there any user interaction possible through UART after receiving this data?

Reply:
No